packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -21,11 +21,17 @@ function getRLCResponseType(operation: Operation) {
     new Set(operation.responses.flatMap((r) => r.statusCodes)).values()
   );
 
-  statusCodes.push("Default");
+  if (hasRLCDefaultResponse(operation)) {
+    statusCodes.push("Default");
+  }
```

Comment:
I don't think we need this if you merge main, we have already mapped the all rlc responses type to modular operations.